src/vendor_upstream/core/globalThisPolyfill.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule globalThisPolyfill
+ */
+
+const globalThis =
+  typeof globalThis !== 'undefined'
+    ? globalThis
```

Comment:
How this statement is working ?, Directly executing this command in console throws error -: "globalThis is not defined".

Reply:
Good catch. FWIW, this works in compiled code since it uses `var` instead of `const`.
I'll fix this.

Reply:
Fixed